l10n/en/404.ftl

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/404/
+
+not-found-page-not-found-page-page-not-found = 404: Page Not Found
+not-found-page-sorry-we-cant-find-that-page = Sorry, we can’t find that page
+not-found-page-were-all-about-a-healthy-internet = We’re all about a healthy internet but sometimes broken URLs happen.
+not-found-page-go-back = Go Back
+
+# Variables:
+#   $about (url) - link to https://www.mozilla.org/about/
+not-found-page-learn-about-mozilla-the-non = <a href="{ $about }">Learn</a> about { -brand-name-mozilla }, the not-for-profit behind { -brand-name-firefox }.
+
+# Variables:
+#   $explore (url) - link to https://www.mozilla.org/firefox/
+not-found-page-explore-the-entire-family-for = <a href={ $explore }>Explore</a> the entire family for { -brand-name-firefox } products designed to respect your privacy.
```

Comment:
Nit: I think this should be "of" instead of "for," so it reads "Explore the entire family **of** Firefox products." 

Though now with VPN I wonder if this should be "Mozilla products" or even just "the entire family of products"? But maybe this isn't the time for a rewrite.

Reply:
Makes sense yeah. For now, I think it's probably best to leave this until when we next update this page (strings have already been reviewed on the l10n side).